IBM_CLOUD_DEPLOY.md

```diff
@@ -0,0 +1,29 @@
+# Deploying to IBM Cloud
+
+1. Create a cluster using the [IBM Cloud Container Service](https://console.bluemix.net/containers-kubernetes/clusters
+). For this Code Pattern we will only need to use the Free tier which gives us access to a Kubernetes cluster with 1 worker node. (Could take anywhere from 10 minutes to half an hour.)
```

Comment:
I would move the parenthetical statement into a "Note"

Reply:
👍